tests/nlu/classifiers/test_sklearn_classifier.py

```diff
@@ -39,21 +42,20 @@ def test_persist_and_load(
     default_sklearn_intent_classifier: SklearnIntentClassifierGraphComponent,
     default_model_storage: ModelStorage,
     default_execution_context: ExecutionContext,
+    train_and_preprocess: Callable[..., Tuple[TrainingData, List[GraphComponent]]],
 ):
-    pipeline = [
-        {"name": "SpacyNLP", "model": "en_core_web_md"},
-        {"name": "SpacyTokenizer"},
-        {"name": "SpacyFeaturizer"},
-    ]
-    loaded_pipeline = [
-        registry.get_component_class(component.pop("name")).create(
-            component, RasaNLUModelConfig()
-        )
-        for component in copy.deepcopy(pipeline)
-    ]
-    for component in loaded_pipeline:
-        component.train(training_data)
+    spacy_model = SpacyModelProvider.load_model("en_core_web_md")
```

Comment:
optional: there is actually already a `spacy_nlp` fixture we could leverage to avoid loading the model into memory